CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## 0.15.3-dev
+ - [#407](https://github.com/tag1consulting/goose/pull/407) move `GooseMetrics::record_users_per_second()` call to the main loop
```

Comment:
I don't think this is worthy of the Changelog, it shouldn't have any impact on the end user.

Reply:
Removed.